<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDynDialogRobotPath</class>
 <widget class="QDialog" name="QDynDialogRobotPath">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>229</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="comboPath">
   <property name="geometry">
    <rect>
     <x>129</x>
     <y>9</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Choose the Scene's path that is going to be followed.</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinSpeed">
   <property name="geometry">
    <rect>
     <x>129</x>
     <y>100</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>This value specifies the linear speed of the frame of reference attached to the selected link.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="accelerated">
    <bool>true</bool>
   </property>
   <property name="suffix">
    <string> m/s</string>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="minimum">
    <double>0.000100000000000</double>
   </property>
   <property name="value">
    <double>0.010000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>137</x>
     <y>146</y>
     <width>81</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>  Cancel</string>
   </property>
   <property name="icon">
    <iconset resource="../Resources.qrc">
     <normaloff>:/resources/icons/cancel.png</normaloff>:/resources/icons/cancel.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ok">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>146</y>
     <width>81</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>   OK    </string>
   </property>
   <property name="icon">
    <iconset resource="../Resources.qrc">
     <normaloff>:/resources/icons/ok.png</normaloff>:/resources/icons/ok.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>122</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="comboLink">
   <property name="geometry">
    <rect>
     <x>129</x>
     <y>40</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>With this list you can specify the Link of the selected Robot which is going to follow the Path specified.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>9</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Path to Follow:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>40</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Link following Path:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>70</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Approach Speed</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>100</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Speed:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinApproach">
   <property name="geometry">
    <rect>
     <x>129</x>
     <y>70</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>This value specifies the linear speed of the frame of reference attached to the selected link.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="accelerated">
    <bool>true</bool>
   </property>
   <property name="suffix">
    <string> m/s</string>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="minimum">
    <double>0.000100000000000</double>
   </property>
   <property name="value">
    <double>0.100000000000000</double>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../Resources.qrc"/>
 </resources>
 <connections/>
</ui>
